wooz1ewu/ITechArt_Front-End
# Sign in/up/out systems implemented. Redirecting added

# Overview  
> Sign in, sign up, sign out processes implemented. Redirecting to login/register pages added.

# Changes  

> 1. Sign-in page added.  
> 1. Sign-in process implemented.  
> 1. Redirecting to login page while user is not logged implemented.  
> 1. Info snackbars added.  
> 1. Sign-up page added. 
> 1. Sign-up process implemented.  
> 1. Form validation added.  
> 1. Log out process implemented.  
> 1. Log out button added to MyNotes page.  

# How to test  

> 1. Visit [sign-up page](https://wooz1ewu.github.io/ITechArt_Front-End/sign-up).  
> 1. Check if all the fields are validated.
> 1. Check if it possible to visit shared notes or my notes pages without signing-in.    
> 1. Try to sign up.
> 1. Check if app redirects you to my-notes page after successful signing-up.  
> 1. Try to edit notes and save it.  
> 1. Try to log out by clicking on log out button on my notes page.  
> 1. Try to sign in using your account via login page.
